src/ipc/persistedStore.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import Store from 'electron-store';
+
+import { SourceName } from './sources';
+
+export type ShownStates = {
+    downloadable: boolean;
+    installed: boolean;
+};
+
+export type WindowState = {
```

Comment:
Is it necessary to export this one?

Reply:
No, it wasn't. :-) Fixed this and more serious issues in a new version which is now pushed.